resources/lang/pt-br/users.php

```diff
@@ -20,17 +20,17 @@
 
 return [
     'login' => [
-        '_' => 'Entrar',
+        '_' => 'Iniciar sessão',
         'username' => 'Nome de usuário',
         'password' => 'Senha',
-        'button' => 'Entrar',
+        'button' => 'Iniciar sessão',
         'remember' => 'Lembrar deste computador',
-        'title' => 'Faça login para continuar',
-        'failed' => 'Login incorreto',
-        'register' => 'Não tem uma conta de osu!? Crie uma',
-        'forgot' => 'Esqueceu sua senha?',
+        'title' => 'Inicie a sessão para continuar',
+        'failed' => 'Nome de usuário e/ou senha incorretos',
+        'register' => 'Não tem uma conta osu!? Crie uma',
```

Comment:
add a "no" between "conta" and "osu!?" should be nice. Would also be better translated.